config/environment.js

```diff
@@ -2,6 +2,13 @@
 
 module.exports = function(environment) {
   var ENV = {
+    apiUrl: 'https://razerapi.herokuapp.com/ryan/basic/brands',
+
+    DS: {
+      host: 'http://localhost:3333',
```

Comment:
Be sure to set the host to `https://razerapi.herokuapp.com` so that the Ember data adapter connects to the heroku api instead of trying to run on your local server.